src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -152,12 +194,13 @@ const AppWrapper = (props: { children: ReactNode }) => {
 
     try {
       await AriesAgent.agent.start();
+      setIsOffline(false);
     } catch (e) {
       // @TODO - foconnor: Should specifically catch the error instead of all, but OK for now.
-      setAgentInitErr(true);
```

Comment:
We can remove this state variable

Reply:
Also - further down in the file, when `agentInitErr` is true, we were displaying a message.

Can we maybe:
1. update it so that it displays while `isOffline` is true,
2. update the message to say "Connecting to the cloud...",
3. Move that to i18n en.json file (ask Sal where to place it)

Will need to discuss this with the team but this can be a temp solution to it.

Reply:
If we display the message, users won't be able to view the app and see the list of identifiers/credentials. Should we turn the message to a smaller component and display it with the identifiers/credentials?

Reply:
...of course :D disregard this comment but can you remove that entire block of code and the comment anyway since we won't need it anymore